### PR TITLE
profiles: mask net-print/{apsfilter,kyocera-mita-ppds} for future last-rite

### PR DESCRIPTION
**Pull request by scardracs**
Closes: https://bugs.gentoo.org/885605
Closes: https://bugs.gentoo.org/884881
Closes: https://bugs.gentoo.org/884307
Closes: https://bugs.gentoo.org/741102

Signed-off-by: Marco Scardovi <mscardovi@icloud.com>

### CODE REVIEW

**Comment by scardracs** [comment]
@thesamesam could you please merge it? I'm gonna post it over gentoo-dev ml, if required

**Comment by scardracs** [comment]
> ## Pull request CI report
> 
> _Report generated at_: 2022-12-14 20:23 UTC _Newest commit scanned_: [4f825be](https://github.com/gentoo/gentoo/commit/4f825be1902dc4dfcd61130bb99d5fbfa1cbdf1e) _Status_: x **broken**
> 
> New issues caused by PR: https://qa-reports.gentoo.org/output/gentoo-ci/84bd336a48/output.html#global
> 
> There are existing issues already. Please look into the report to make sure none of them affect the packages in question: https://qa-reports.gentoo.org/output/gentoo-ci/84bd336a48/output.html

Nevermind, I've already fixed it

File: profiles/package.mask [hunk 1]
```diff
@@ -33,6 +33,20 @@
...
+# an installation option for that package.
+# It still uses EAPI 6 and no real support since
+# years.
+# Removal on 2023-01-14. Bug #885605
+net-print/kyocera-mita-ppds
+
+# Marco Scardovi <mscardovi@icloud.com> (2022-12-14)
+# Dead upstream, dead homepage, dead symlinks,
+# no revdeps and no version bump since 2011
+# Removal on 2023-01-14. Bug #884307
```
Comment (juippis) [inline profiles/package.mask:47]: Let's use bug \#884881.

**Review by juippis — APPROVED** [review]
We can't "close" the bugs with a package.mask, if this mask was reverted the bugs would be closed even though they are valid. I'll switch to use a `Bug:` reference instead.

(For a public note:) Please send out the last-rites only after the mask has been applied.

**Pull request closed without merging**